src/gui/inputdevices/TouchInputHandler.h

```diff
@@ -35,6 +35,8 @@ class TouchInputHandler: public AbstractInputHandler {
     utl::Point<double> priLastRel{-1.0, -1.0};
     utl::Point<double> secLastRel{-1.0, -1.0};
 
+    bool startZoomReady = false;
```

Comment:
Please add a comment that describes what this variable is for. This way one must not first read the full code to know what it does.